client/public/manifest.json

```diff
@@ -3,13 +3,14 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "favicon.png",
       "sizes": "192x192",
```

Comment:
@MrDwina you also need to change the size to 512x512

Reply:
ok alright